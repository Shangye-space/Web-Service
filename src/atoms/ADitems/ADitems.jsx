import React from 'react';

import styles from './ADitems.module.scss';

const ADitems = () => {
    return (
        <div className={styles.items}>
            <div className={styles.left}>
                <img src="/images/IMG_2592.jpg" />
            </div>
            <div className={styles.right}>
                <img src="/images/RXTM-088-600x600.jpg" />
            </div>
        </div>
    );
};

export default ADitems;
