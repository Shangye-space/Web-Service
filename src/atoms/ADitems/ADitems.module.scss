.items {
    display: flex;
    flex-direction: row;
    height: 240px;

    div {
        height: 100%;

        img {
            height: 100%;
            border-radius: 2%;

            filter: brightness(90%);
        }
    }
}

.left {
    padding-right: 15px;
}

.right {
    padding-left: 15px;
}
