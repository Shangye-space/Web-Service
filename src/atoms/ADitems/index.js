import ADitems from './ADitems.jsx';

export default ADitems;
