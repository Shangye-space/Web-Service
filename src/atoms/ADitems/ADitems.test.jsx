import React from 'react';
import ReactDOM from 'react-dom';
import ADitems from './ADitems.jsx';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ADitems />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('snapshot renders', () => {
    const component = renderer.create(<ADitems />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
