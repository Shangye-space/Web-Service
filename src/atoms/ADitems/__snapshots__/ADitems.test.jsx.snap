// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot renders 1`] = `
<div
  className="items"
>
  <div
    className="left"
  >
    <img
      src="/images/IMG_2592.jpg"
    />
  </div>
  <div
    className="right"
  >
    <img
      src="/images/RXTM-088-600x600.jpg"
    />
  </div>
</div>
`;
